source/components/JsxParser.tsx

```diff
@@ -30,6 +30,15 @@ export type TProps = {
 }
 type Scope = Record<string, any>
 
+interface ForwardRefComponentProps {
```

Comment:
This doesn't seem terribly useful as a type. Probably better just to use `any` or `unknown` here instead of essentially just defining a `Record`